backend/gene_info/api/ncbi_provider.py

```diff
@@ -62,11 +62,15 @@ def fetch_gene_uid(self, geneID, gene):
             raise NCBIUnexpectedResultException
 
         # search with gene name if needed
-        if not self._is_valid_search_result(search_response) and gene != "":
+        if not self._is_valid_search_result(search_response) and gene and gene != "":
             try:
-                search_response = self._search_gene_uid(gene)
+                term = "(" + str(gene) + "%5BGene%20Name%5D)%20AND%20human%5BOrganism%5D"
```

Comment:
can you pull `%5BGene%20Name%5D)%20AND%20human%5BOrganism%5D` out as a const and reuse it in the tests?